CONTRIBUTING.md

```diff
@@ -184,6 +184,15 @@ npm run build
 npm run test:compat
 ```
 
+> [!NOTE]
+> On Node.js v18 this may fail. If this happens, change the `.mocharc.yml` file
+> as follows:
+>
+> ```diff
+> - import: tsx
+> + require: tsx
+> ```
+
```

Comment:
This should be reverted as the note is outdated following d29d421d3317c284bbd8f4441755dbe769e054d3

Reply:
Resolved with e27087433767c4f0fc38ab38ac4720bc50c573ae